pyflakes/test/test_asserts.py

```diff
@@ -0,0 +1,39 @@
+from pyflakes import messages as m
+from pyflakes.test.harness import TestCase
+
+
+class Test(TestCase):
+    def test_Asserts_intentionally_simple_pass(self):
+        self.flakes("""
+        assert True
+        assert False
+        """)
+
+    def test_Asserts_nontrivial_pass(self):
+        self.flakes("""
+        d = {'a': 1, 'b': 2}
+        l = [1, 2]
+        n = 10
+        s = {1, 2}
+        t = (1, 2)
+        needle = 'needle'
+        assert d == {'a': 3}
+        assert n == 10
+        assert l == [1, 2]
+        assert s == {1, 2}
+        assert t == (1, 2)
+        assert needle in "haystack"
```

Comment:
Can you add a test for something like

```python
assert conditional, "Message in case of assertion error"
```

That would be an implicit tuple if the assert statement didn't handle it differently (and tuples are tested below)